maintainers/maintainer-list.nix

```diff
@@ -8417,6 +8417,12 @@
     githubId = 997855;
     name = "Narazaki Shuji";
   };
+  shofius = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add shofius``.

Reply:
done